src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -50,5 +63,9 @@ public CacheOptions(bool useSharedCache)
         /// </remarks>
         public bool UseSharedCache { get; set; }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public IEnumerable<string> ExtraTokenParamsToCache { get; set; }
```

Comment:
Naming - "ExtraPayloadParameters" to be consistent with "ExtraQueryParameters" ? (and just a comment that it's only applied to /token). 

Please involve @localden in all public API discussions.